e2e/runner/start-sample-app-containers/start.ts

```diff
@@ -15,28 +15,30 @@ import type {
 
 const userOptions = {
   EMBEDDING_SDK_VERSION: "local",
+  SAMPLE_APP_BRANCH_NAME: "",
   ...process.env,
 } as const;
 
 printBold(`Running Cypress Sample App Tests with options:
   - EMBEDDING_SDK_VERSION      : ${userOptions.EMBEDDING_SDK_VERSION}
+  - SAMPLE_APP_BRANCH_NAME     : ${userOptions.SAMPLE_APP_BRANCH_NAME}
 `);
 
-export async function startSampleAppContainers(
-  testSuite: SampleAppTestSuiteName,
-) {
+export async function start(testSuite: SampleAppTestSuiteName) {
```

Comment:
imo, I think `startSampleAppContainers` is nice and explicit - not sure why this is renamed